packages/manager/modules/pci/src/error/error.routing.js

```diff
@@ -0,0 +1,97 @@
+import get from 'lodash/get';
+import snakeCase from 'lodash/snakeCase';
+
+import EnvironmentService from '@ovh-ux/manager-config';
+
+import { PCI_REDIRECT_URLS } from '../constants';
+import {
+  IMAGE_ASSETS,
+  PAY_DEBT_URL,
+} from './error.constants';
+
+
+export default /* @ngInject */ ($stateProvider) => {
+  $stateProvider
+    .state('pci.error', {
+      url: '/error',
+      component: 'pciProjectError',
+      params: {
+        code: {
+          type: 'any',
+        },
+        context: {
+          type: 'any',
+        },
+        message: {
+          type: 'any',
+        },
+      },
+      resolve: {
+        breadcrumb: () => null,
+        error: /* @ngInject */ ($transition$) => {
+          const stateParams = $transition$.params();
+          return {
+            code: snakeCase(stateParams.code),
+            context: stateParams.context,
+            message: stateParams.message,
+          };
+        },
+        getActionButtonHref: () => (errorCode) => {
+          switch (errorCode) {
+            case 'unpaid_debts':
```

Comment:
Possible to make constants ? :)

Reply:
Noted. It is planned to change that logic, it will be done at that moment.